Jellyfin.Api/Controllers/ItemUpdateController.cs

```diff
@@ -224,7 +224,7 @@ public ActionResult UpdateItemContentType([FromRoute, Required] Guid itemId, [Fr
             return NoContent();
         }
 
-        private void UpdateItem(BaseItemDto request, BaseItem item)
+        private async void UpdateItem(BaseItemDto request, BaseItem item)
```

Comment:
Don't use `async void`

Reply:
What would I use instead? Don't I need to await the saving on the item repo?

Reply:
Yes, but this function needs to be `async Task`